tests/PartnerClientTest.php

```diff
@@ -65,6 +66,29 @@ public function testChangeSeatsHappyPath()
             self::fail('error on ChangeSeats: ' . $e);
         }
 
-        self::assertNotEmpty($resp, 'expected a response');
+        self::assertEmpty($resp, 'expected an empty response');
+    }
+
+    public function testUpdateSSOHappyPath()
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $partnerClient = new PartnerClient($environment);
+
+        $req = new UpdateSSORequest();
+        $req->setDomain("google.com");
+        // disable SSO
+        $req->setEnableSso(false);
+
+        try {
+            $resp = $partnerClient->UpdateSSO($req);
+        } catch (Vendasta\Vax\SDKException $e) {
+            self::assertEquals(404, $e->getCode(), 'expected a 404');
```

Comment:
Imo we don't want to pass on a 404 for this test, I think this should fail the test